docs/reference/zedtokens-and-zookies.md

```diff
@@ -72,41 +72,42 @@ CheckPermissionRequest{
 }
 ```
 
-[the Check response]: https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.CheckPermissionResponse
-[the Write response]: https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.WriteRelationshipsResponse
-[v1 API]: https://buf.build/authzed/api/tree/main/authzed/api/v1
-[Consistency]: api-consistency.md
+[the check response]: https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.CheckPermissionResponse
+[the write response]: https://buf.build/authzed/api/docs/main:authzed.api.v1#authzed.api.v1.WriteRelationshipsResponse
+[v1 api]: https://buf.build/authzed/api/tree/main/authzed/api/v1
+[consistency]: api-consistency.md
 
 ## Frequently Asked Questions
 
 ### If I store ZedTokens will they become stale at some point?
 
-In v1, the recommendation is to use `at_least_as_fresh` in the [Consistency] option passed to your `CheckPermission` call, ensuring that it defines merely a lower bound for staleness.
+No. Since a ZedToken represents a _point-in-time_ after which caches must be caught up, if you use an older ZedToken with `at_least_as_fresh` in the [Consistency] option passed to your `CheckPermission` call, the ZedToken ensures that it defines merely a lower bound for staleness.
```

Comment:
Maybe we should mention that it does go stale for `at_exact_snapshot` when it leaves the datastore gc window